src/math/Triangle.js

```diff
@@ -106,6 +107,29 @@ Object.assign( Triangle, {
 
 		};
 
+	}(),
+
+	uvIntersection: function () {
+
+		var barycoord = new Vector3();
+		var v1 = new Vector2();
+		var v2 = new Vector2();
+		var v3 = new Vector2();
+
+		return function uvIntersection( point, p1, p2, p3, uv1, uv2, uv3, result ) {
+
+			this.getBarycoord( point, p1, p2, p3, barycoord );
+
+			v1.copy( uv1 ).multiplyScalar( barycoord.x );
+			v2.copy( uv2 ).multiplyScalar( barycoord.y );
+			v3.copy( uv3 ).multiplyScalar( barycoord.z );
+
+			v1.add( v2 ).add( v3 );
+
+			return result.copy( v1 );
+
+		};
+
```

Comment:
How about naming the method `getUV()`. You can simplify the code like so:
```javascript
getUV: function () {

	var barycoord = new Vector3();

	return function getUV( point, p1, p2, p3, uv1, uv2, uv3, target ) {

		this.getBarycoord( point, p1, p2, p3, barycoord );

		target.set( 0, 0 );
		target.addScaledVector( uv1, barycoord.x );
		target.addScaledVector( uv2, barycoord.y );
		target.addScaledVector( uv3, barycoord.z );

		return target;

	};

}()
```

Reply:
@06wj I see you used this code.  Did you test it?

Reply:
Yes, I test it in [webgl_raycast_sprite](https://rawgit.com/06wj/three.js/patch1_build/examples/webgl_raycast_sprite.html) and [webgl_raycast_texture](https://rawgit.com/06wj/three.js/patch1_build/examples/webgl_raycast_texture.html)

Reply:
Great!